vital/types/geodesy.h

```diff
@@ -76,12 +76,16 @@ namespace SRID
 class geo_conversion
 {
 public:
+  virtual char const* id() const = 0;
```

Comment:
*Could* it be? Yes, certainly.

*Should* it be? I tend to think not. As-is, the intent is for this to return a literal in `.rodata`. That isn't possible with a `std::string`, which would have to be constructed as a static object, for questionable benefit. (If you have a good reason why the user needs a `std::string`, I might change my mind.)

I actually thought about introducing a `kwiver::string_view` for this... if we had `string_view`, that would be the clear winner.